h/static/scripts/directive/annotation-share-dialog.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var angular = require('angular');
+
+module.exports = function () {
+  return {
+    bindToController: true,
+    controllerAs: 'vm',
+    restrict: 'E',
+    template: require('../../../templates/client/annotation_share_dialog.html'),
+    controller: ['$timeout', '$scope', function ($timeout, $scope) {
```

Comment:
It should be possible to use an `// @ngInject` comment here rather than using the array syntax, eg.

```js
// @ngInject
controller: function ($scope) {
...
}
```

The `$timeout` variable is no longer used.